app/src/main/java/br/com/hay/splash/SplashPresenter.kt

```diff
@@ -0,0 +1,10 @@
+package br.com.hay.splash
+
+class SplashPresenter : SplashContract.Presenter {
```

Comment:
Please, consider removing the SplashPresenter because the splash screen will not use MPV.